### PR TITLE
Fix track repeat bug

### PR DESCRIPTION
**Pull request by jafayer**
Is tested on Android and iOS

### CODE REVIEW

File: ios/LanguageTransfer/Info.plist [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>10</string>
```
Comment (SyntaxBlitz) [inline ios/LanguageTransfer/Info.plist:24]: this was probably being set by testflight before? let's just keep an eye on this

File: js/audio-service.ts [hunk 2]
```diff
@@ -286,7 +293,7 @@ export default async () => {
           wasPlaying.course,
           wasPlaying.lesson,
         );
-
+
```
Comment (SyntaxBlitz) [inline js/audio-service.ts:296]: no

File: ios/LanguageTransfer/Info.plist (hunk 1 above)
Comment (jafayer) [reply ios/LanguageTransfer/Info.plist:24]: Yeah that's my guess, I didn't change that

Comment (jafayer) [reply ios/LanguageTransfer/Info.plist:24]: oh that's how I interpreted it anyway

**Pull request merged**